lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -48,6 +48,12 @@ def setup_rabbitmq_connection_config
         # Time in seconds to wait before retrying a connection
         config :connect_retry_interval, :validate => :number, :default => 1
 
+        # The default connection timeout; zero means wait indefinitely
+        config :connection_timeout, :validate => :number
```

Comment:
similar here as the default. "If not specified, the connection will not timeout ever - and that may be OK for you" or something